dsaps/helpers.py

```diff
@@ -48,21 +42,21 @@ def create_metadata_id_list(metadata_csv):
     return metadata_ids
 
 
-def match_files_to_metadata(file_dict, metadata_ids):
+def match_files_to_metadata(file_list, metadata_ids):
```

Comment:
See comment below for `match_metadata_to_files`, re: both the function and the test.